CONTRIBUTORS.md

```diff
@@ -20,5 +20,6 @@
 - [Quentin Bey](https://github.com/qbey) <quentin.bey@polyconseil.fr>
 
 - [Renovate Bot](https://renovatebot.com) <bot@renovateapp.com>
+- [Renovate Bot](https://renovatebot.com) <29139614+renovate[bot]@users.noreply.github.com>
```

Comment:
Wow this looks like a trap. Pretty sure we will have a different mail every week. I don't know if should not ignore renovate emails.

Reply:
We can give it a try and next week if the mail has changed we will create an ignore list for dependabot and renovate.